tests/functional_tests/ffront_tests/test_execution.py

```diff
@@ -44,6 +44,11 @@
 fieldview_backend = roundtrip.executor
 
 
+@pytest.fixture(params=[roundtrip.executor, gtfn_cpu.run_gtfn])
+def fieldview_backend_list(request):
```

Comment:
why is this named `fieldview_backend_list`? Does it yield a `list`? If so, how do the decorators accept lists as backends?

Reply:
Do you have any suggestions?